spinalcordtoolbox/deepseg/models.py

```diff
@@ -443,6 +446,9 @@ def is_up_to_date(path_model):
     if model_name not in MODELS:
         logger.warning(f"Model name '{model_name}' from source.json does not match model names in SCT source code.")
         return False
+    if "custom" in source_dict and source_dict["custom"] is True:
+        return True  # Don't reinstall the model if the 'custom' flag is set (since custom URLs would fail comparison)
```

Comment:
Maybe print a warning that a custom model is being used? Ideally with the URL itself (unless it already appears nearby in the logging output), and a command that can be run to revert to the official model (I guess that's just the `-install` command with the task name, since it doesn't check `is_up_to_date()`).

Otherwise I imagine people will forget that they customized it, and I don't think anyone will pay attention to the actual URL even if it appears in the output.